source/includes/fundamentals/code-snippets/libmongocrypt-maven-versioned.rst

```diff
@@ -0,0 +1,10 @@
+.. code-block:: sh
```

Comment:
Nitpick: you can use "xml" here, similar to this https://raw.githubusercontent.com/mongodb/docs-java/master/source/includes/quick-start/maven-versioned.rst